hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveViewCatalog.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iceberg.hive;
+
+import java.util.Collections;
+import org.apache.iceberg.catalog.Catalog;
+import org.apache.iceberg.view.ViewCatalogTests;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+
+public class TestHiveViewCatalog extends ViewCatalogTests<HiveCatalog> {
+
+  private HiveCatalog catalog;
+
+  @BeforeEach
+  public void before() throws Exception {
+    HiveMetastoreTest.startMetastore(Collections.emptyMap());
```

Comment:
this should use the JUnit5 extension from https://github.com/apache/iceberg/pull/8918 rather than using `HiveMetastoreTest` directly